### PR TITLE
Kuz 404 content type error

### PR DESCRIPTION
**Pull request by dbengsch**
Treate content-type error before anything else.

### CODE REVIEW

File: lib/api/core/models/repositories/repository.js [hunk 1]
```diff
@@ -341,6 +341,7 @@ Repository.prototype.hydrate = function (object, data) {
  * Persists the given object in the collection that is attached to the repository.
  *
  * @param {ObjectConstructor} object The object to persist
+ * @param {Object} opts The peristence options
```
Comment (scottinet) [inline lib/api/core/models/repositories/repository.js]: nitpicking: `persistence`

**Comment by scottinet** [comment]
+1

**Comment by ballinette** [comment]
:+1:

**Comment by benoitvidis** [comment]
+1

**Pull request merged**